api/apps/graphql/resolvers.js

```diff
@@ -64,6 +64,20 @@ export const authors = (root, args, req, ast) => {
   })
 }
 
+export const relatedAuthors = (root) => {
```

Comment:
Seems kinda funny to call this `relatedAuthors` if we've just been using the term `authors`?

Reply:
Hmm I guess we've already got `authors` as a resolver -- i suppose this is fine!